flow/Knobs.h

```diff
@@ -93,6 +93,7 @@ class FlowKnobs : public Knobs {
 	int64_t SIM_PAGE_CACHE_64K;
 	int64_t BUGGIFY_SIM_PAGE_CACHE_4K;
 	int64_t BUGGIFY_SIM_PAGE_CACHE_64K;
+	int CACHE_EVICTION_POLICY; // 0 RANDOM (default), 1 LRU
```

Comment:
We actually do support std::string knobs, and I think cache eviction policy is a good use for one.  We could add other policies in the future and strings would be the most user-friendly way to specify them.  A member in AsyncFileCached of type CacheEvictionType can then be initialized from the knob's string value.

Reply:
Sounds good! I will change the knob to be a string, add a member (evictionType) and initialize it based on the knob's string value, and update the patch.

Reply:
@satherton updated with the knob being a string.